docs/crio.conf.5.md

```diff
@@ -75,6 +75,9 @@ Location for CRI-O to lay down the clean shutdown file.
 It is used to check whether crio had time to sync before shutting down.
 If not found, crio wipe will clear the storage directory.
 
+**goroutines_monitor**=false
```

Comment:
I wonder how many goroutines we can say is abnormal.
Last time I checked it was ~30 independent from #pods, but not sure...